packages/go_router/lib/src/builder.dart

```diff
@@ -63,6 +64,12 @@ class RouteBuilder {
   /// its history.
   final String? restorationScopeId;
 
+  /// Whether or not the navigator and it's new topmost route should request focus
+  /// when the new route is pushed onto the navigator.
+  /// 
+  /// Defaults to true.
```

Comment:
maybe add a link to navigator.requestfocus

Reply:
done